.github/workflows/on-push.yml

```
@@ -174,7 +174,7 @@ jobs:
         python -m pip install --no-deps -e .
     - name: Run tests
       run: |
-        make unit-tests COV_REPORT=xml
+        make integration-tests COV_REPORT=xml
```

```suggestion
        make unit-tests COV_REPORT=xml
```
Hi @garciampred,

Please revert this change. In general, we control all configuration files with Cookiecutter + Cruft.

`make integration-tests` is not meant to be executed from GitHub CI. We only use it to test development branches and to ensure everything is in good shape before a release. For example, you can run it out-of-the-box from dev-explorer or locally with an appropriate `.cads-api-client.json`.

We use the integration tests on GitHub to test different Python versions and/or environments with different dependencies for packages that have optional dependencies.

---

ok sorry, I was just testing, the PR is not finished

---

OK, sorry about that.
This is a public repo:  You should be able to use Draft PR when you are just tinkering.